### ISSUE TITLE
Reconnect Issue/Bug

### ISSUE THREAD

**Issue by LockHarte98** [ValveSoftware/Dota2-Gameplay#15298]
### Description

I am always getting disconnected whenever confirming a match as soon as getting to the picking-of-heroes phase. Now I am penalized for 1 hour from the error. I cannot reconnect no matter how many times I try and even update and restart. Fix this please! 

### Screenshots

![423568283_364594509768630_3818603312636469256_n](https://github.com/ValveSoftware/Dota2-Gameplay/assets/159529288/64230218-774e-4dae-9240-5bfad6e387cc)

**Comment by Riismannen** [comment]
Yes seeming to be an issue for many right now. I’m kicked out myself. Dota please fix

**Comment by Drolito** [comment]
same here, we are lot of players ... :(

**Comment by fasjkhygfda8yhgfiuqwyehi7u** [comment]
what is the solution guys

**Comment by fasjkhygfda8yhgfiuqwyehi7u** [comment]
any help ?

**Comment by Drolito** [comment]
Solution is to wait for valve correcting this issue ...

**Comment by LockHarte98** [comment]
Most of the time, solution is another update fixing the bug... We just need to wait till they fix it.

**Comment by godspeed2uman** [comment]
I had the issue where i cant connect to a found match and cant spectate; reinstalled the game and now the issue is fixed.

**Comment by Drolito** [comment]
Not dure it works ... For me didnt worked ...
